changelogutils/validator.go

```diff
@@ -307,6 +316,18 @@ func (c *changelogValidator) getValidationSettings(ctx context.Context) (*Valida
 	return GetValidationSettings(ctx, c.code, c.client)
 }
 
+func (c *changelogValidator) GetChangelogDirectory(ctx context.Context) string {
```

Comment:
This code is identical (or very close) to the changelogutil reader.go, what is the difference between the two?